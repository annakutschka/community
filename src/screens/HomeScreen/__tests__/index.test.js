import React from 'react';
import renderer from 'react-test-renderer';
import { HomeScreen } from '../index';

const props = {
  fetchAnnouncements: jest.fn()
};

describe('<HomeScreen />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<HomeScreen {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
