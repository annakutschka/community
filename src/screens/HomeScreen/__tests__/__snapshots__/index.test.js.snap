// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeScreen /> renders correctly 1`] = `
<RCTSafeAreaView
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#f3f3f3",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "flex-start",
    }
  }
/>
`;
